tutorials/2d/using_tilemaps.rst

```diff
@@ -211,25 +211,81 @@ can test for.
 2x2 mode can only match cells that are part of a 2-by-2 block - cells with no
 neighbors and lines only one cell wide are not supported.
 
+**Template - Generic:**
+
+This template can be used for sideways or fully top-down perspectives.
+It's designed for a TileMap cell size of 64x64.
+
+Key:
+
+- Red: "on"
+- White: "off"
+
+.. image:: img/autotile_template_2x2.png
+    :target: ../../_images/autotile_template_2x2.png
```

Comment:
It's weird to have to hardcode a path like that, and it will likely break for localized versions.

Reply:
I needed to do this to make the image clickable (so it can be saved more easily). Otherwise, I guess we should write a small Sphinx extension for this.

Reply:
I'd remove the hardcoded path for now, I guess the image can still be saved with right-click?

Reply:
Done. I added an explicit note about it, just in case.